utils/update.js

```diff
@@ -0,0 +1,30 @@
+const eddsa = require("../circomlib/src/eddsa.js");
+const txLeaf = require("./generate_tx_leaf.js");
+const babyJub = require("../circomlib/src/babyjub");
+const snarkjs = require("snarkjs");
+const bigInt = snarkjs.bigInt;
+const createBlakeHash = require("blake-hash");
+
+module.exports = {
+
+    processTx: function(tx, fromLeaf, toLeaf, signature){
+        const sBuff = eddsa.pruneBuffer(createBlakeHash("blake512").update(prv).digest().slice(0,32));
```

Comment:
We might want to use a hash function that is available in the EVM here as we may need to validate signatures in the EVM at some stage.

Reply:
Probably not in scope to change it in this PR just a note.

Reply:
sorry -- removed this from the utils, should have been using verifyMiMC which doesn't require Blake hash.